src/Swashbuckle.SwaggerGen/Annotations/SwaggerProducesResponseTypeFilter.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+using Swashbuckle.Swagger.Model;
+using Swashbuckle.SwaggerGen.Generator;
+
+namespace Swashbuckle.SwaggerGen.Annotations
+{
```

Comment:
Remove XML comments as per above, it's hard to read the code with all the comments in the way :)